alacritty_terminal/src/tty/unix.rs

```diff
@@ -302,89 +317,85 @@ impl EventedReadWrite for Pty {
     #[inline]
     fn register(
         &mut self,
-        poll: &mio::Poll,
-        token: &mut dyn Iterator<Item = mio::Token>,
-        interest: mio::Ready,
-        poll_opts: mio::PollOpt,
+        poll: &Arc<polling::Poller>,
+        mut interest: polling::Event,
+        poll_opts: polling::PollMode,
     ) -> Result<()> {
-        self.token = token.next().unwrap();
-        poll.register(&EventedFd(&self.file.as_raw_fd()), self.token, interest, poll_opts)?;
+        self.token = PTY_READ_TOKEN;
```

Comment:
Is there any point in this? It'll always be 0 anyway.

Reply:
It clarifies that the token is representative of the read token.

Reply:
How is that any different from just initializing it to the read token from the start?

Reply:
This mirrors how it was done in the `mio` code. If you want I can change it to be initialized in this way from the start.

Reply:
Would probably be cleaner this way. The mio code wasn't perfect, there's no need to keep things consistent for the sake of being similar to the mio code when there's a better solution available.